typescript-loops-practice/get-indexes.ts

```diff
@@ -0,0 +1,8 @@
+/* exported getIndexes */
+function getIndexes(array: unknown[]): unknown[] {
+  const newArray = [];
+  for (let i = 0; i < array.length; i++) {
+    newArray.push(array.indexOf(array[i]));
+  }
+  return newArray;
+}
```

Comment:
You are doing a bit too much work here on line 5. We just want to add the `index` to our `newArray`. What part of our loop is representing our `index`?